src/core/lib/iomgr/error.cc

```diff
@@ -405,6 +405,9 @@ static grpc_error* copy_error_and_unref(grpc_error* in) {
       internal_set_str(&out, GRPC_ERROR_STR_DESCRIPTION,
                        grpc_slice_from_static_string("cancelled"));
       internal_set_int(&out, GRPC_ERROR_INT_GRPC_STATUS, GRPC_STATUS_CANCELLED);
+    } else if (in == GRPC_ERROR_EOS) {
+      internal_set_str(&out, GRPC_ERROR_STR_DESCRIPTION,
+                       grpc_slice_from_static_string("end of stream"));
```

Comment:
Might be a good idea to change this to use `eos_error_string`, defined below.

Same thing for the previously existing cases here.

Reply:
I'm not sure what the reason was to do it this way, but the difference between the two is the escaped quotes at the beginning and the end of the string.